pincer/client.py

```diff
@@ -366,11 +366,12 @@ async def execute_event(self, call: Coro, *args, **kwargs):
         :param \\*kwargs:
             The named arguments for the event.
         """
+        def execute(*_args, **_kwargs):
+            ensure_future(call(*_args, **_kwargs))
 
-        if should_pass_cls(call):
-            await call(self, *args, **kwargs)
-        else:
-            await call(*args, **kwargs)
+        execute(self, *args, **kwargs) \

```

Comment:
Can you change the style of this bloc?

I feel like the actual way it is written could be improved to not have backslashes.

I tend to use ternary only when using a `return` or assigning to a variable, cause i find the classic if/else more readable otherwise, what do you think?